tests/test_waterwheel.py

```diff
@@ -210,7 +275,7 @@ def test_province_abbreviations(self):
 
         for ent in doc.ents:
             self.assertIsNotNone(ent._.wikilink)
-    
+
     def test_nonalphabetical_matches(self):
         doc = self.nlp('There is no waterbody in this (), ( ) sentence.')
         self.assertEqual(len(doc.ents), 0)
```

Comment:
Please add countries, mountains also to this. Once the doc_bin is reformed, we need to make changes in `waterwheel.py` as well for the new vocab to take effect.